### PR TITLE
filter-map-view

### PR DESCRIPTION
**Pull request by resmaa**
**Release Feature 3.1**

Map view filter
PEM 455

### CODE REVIEW

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1054#" title="Author self-approved">resmaa</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: content/docs/04-clusters/06-cluster-management/14-noc-ui.md
Comment (karl-cardenas-coding) [inline content/docs/04-clusters/06-cluster-management/14-noc-ui.md:28]:
```suggestion
* Select the cluster to which the location needs to be updated and go to the **Cluster details** page of the cluster.
```

File: content/docs/04-clusters/06-cluster-management/14-noc-ui.md
Comment (karl-cardenas-coding) [inline content/docs/04-clusters/06-cluster-management/14-noc-ui.md:31]:
```suggestion
* Open **Settings** and then navigate to the **Cluster Settings**.
```

File: content/docs/04-clusters/06-cluster-management/14-noc-ui.md
Comment (karl-cardenas-coding) [inline content/docs/04-clusters/06-cluster-management/14-noc-ui.md:34]:
```suggestion
* Select **Location** from the left menu, set the cluster's location, and save the changes.
```
"To that wizard" sounds awkward because although we are using a wizard, it's not what we save the changes to. The wizard is just a UI tool to help the user.

File: content/docs/04-clusters/06-cluster-management/14-noc-ui.md
Comment (karl-cardenas-coding) [inline content/docs/04-clusters/06-cluster-management/14-noc-ui.md:47]:
```suggestion
* Log in to Palette and select **Clusters** from the left **Main Menu**.
```

File: content/docs/04-clusters/06-cluster-management/14-noc-ui.md
Comment (karl-cardenas-coding) [inline content/docs/04-clusters/06-cluster-management/14-noc-ui.md:50]:
```suggestion
* Go to **Map View Icon** below the **Clusters** tab.
```
Clusters is a UI element so it needs bold.

File: content/docs/04-clusters/06-cluster-management/14-noc-ui.md
Comment (karl-cardenas-coding) [inline content/docs/04-clusters/06-cluster-management/14-noc-ui.md:57]:
```suggestion
Palette Map Filters filter out specific clusters using built-in or custom filters for an enhanced user experience. The map filter allows you to narrow down clusters that may be dispersed geographically, across multiple scopes, or different cloud providers. You have two types of filters:- **Built-in Filters** and **Custom Filters**.
```

Vale is correct in this feedback, we should not use plurals in parentheses.

File: content/docs/04-clusters/06-cluster-management/14-noc-ui.md
Comment (karl-cardenas-coding) [inline content/docs/04-clusters/06-cluster-management/14-noc-ui.md:59]:
```suggestion
## Built-In Filters
```
Style guide ^

File: content/docs/04-clusters/06-cluster-management/14-noc-ui.md
Comment (karl-cardenas-coding) [inline content/docs/04-clusters/06-cluster-management/14-noc-ui.md:61]:
```suggestion
Built-in filters are available in the Palette console by default and can be selected from the **Add Filter** drop-down menu. You can use the following built-in filters.
```
This re-writes the sentence by specifying in the beginning what is available. The second sentence is re-written to active voice.

File: content/docs/04-clusters/06-cluster-management/14-noc-ui.md
Comment (karl-cardenas-coding) [inline content/docs/04-clusters/06-cluster-management/14-noc-ui.md:68]: We should explain what each filter is for. Let's add a description column.

File: content/docs/04-clusters/06-cluster-management/14-noc-ui.md
Comment (karl-cardenas-coding) [inline content/docs/04-clusters/06-cluster-management/14-noc-ui.md:76]:
```suggestion
* Log in to Palette and select **Clusters** from the left **Main Menu**.
```

File: content/docs/04-clusters/06-cluster-management/14-noc-ui.md
Comment (karl-cardenas-coding) [inline content/docs/04-clusters/06-cluster-management/14-noc-ui.md:90]: This does not need an infobox. This can simply be part of the information before the table.

File: content/docs/04-clusters/06-cluster-management/14-noc-ui.md
Comment (karl-cardenas-coding) [inline content/docs/04-clusters/06-cluster-management/14-noc-ui.md:87]:
```suggestion
* You can add more than one custom filter simultaneously, and they work together with the chosen conjunction.
```

File: content/docs/04-clusters/06-cluster-management/14-noc-ui.md
Comment (karl-cardenas-coding) [inline content/docs/04-clusters/06-cluster-management/14-noc-ui.md:113]:
```suggestion
```
We shouldn't use a markdown table for styling purposes.

**Review by karl-cardenas-coding — CHANGES_REQUESTED** [review]
Feedback and suggestions are below.

**Comment by spectro-prow** [comment]
@resmaa: The following test **failed**, say `/retest` to rerun all failed tests:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
librarium_build | 0824ec4a8b0bf3e5dad63e0d4e545d6bff6d6adf | [link](http://34.120.121.77/view/gcs/capi-prow-artifacts/pr-logs/pull/spectrocloud_librarium/1054/librarium_build/1604982161096577024) | `/test librarium_build`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

File: content/docs/04-clusters/06-cluster-management/14-noc-ui.md
Comment (karl-cardenas-coding) [inline content/docs/04-clusters/06-cluster-management/14-noc-ui.md:18]:
```suggestion
Palette provides intuitive UI-based location monitoring for the clusters running at multiple locations. Palette displays the region set during the cluster creation process for public cloud clusters and the location on the UI Map. The user can set the location for private cloud clusters through the Palette UI. The user can monitor the location details of all the clusters running under a specific scope. 
```
Minor cleanup

File: content/docs/04-clusters/06-cluster-management/14-noc-ui.md
Comment (karl-cardenas-coding) [inline content/docs/04-clusters/06-cluster-management/14-noc-ui.md:25]:
```suggestion
* Login to [Palette](https://console.spectrocloud.com).
```

**Review by karl-cardenas-coding — APPROVED** [review]

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1054#pullrequestreview-1226305911" title="Approved">karl-cardenas-coding</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/1054#" title="Author self-approved">resmaa</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/release-3-1/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: content/docs/04-clusters/06-cluster-management/14-noc-ui.md
Comment (karl-cardenas-coding) [inline content/docs/04-clusters/06-cluster-management/14-noc-ui.md:18]:
```suggestion
Palette provides an intuitive user interface (UI) based on location monitoring for the clusters running at multiple locations. The Palette 
UI displays the region set during the cluster creation process for public cloud clusters and the location on the UI map. You can set the location for private cloud clusters through the Palette UI. You can also monitor the location details of all the clusters running under a specific scope. 
```

**Pull request merged**
